### PR TITLE
add tempita

### PR DESCRIPTION
**Pull request by msarahan**

### CODE REVIEW

**Comment by jakirkham** [comment]
Could you please rebase on `master` again @msarahan? I think you picked up the jinja recipe by accident.

**Comment by jakirkham** [comment]
Thanks.

**Comment by msarahan** [comment]
wow!  there's a narrow window for that!  That's like winning the lottery.  I'll go buy a ticket.

File: recipes/tempita/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  summary: A small templating language
+  description: |
+    tempita is a small templating language. This language implements
+    if/elif/else, for/continue/break, expressions, and blocks of Python code.
+  doc_url: http://pythonpaste.org/tempita/
+  dev_url: https://bitbucket.org/ianb/tempita
+
+extra:
+  recipe-maintainers:
+    - msarahan
```
Comment (jakirkham) [inline recipes/tempita/meta.yaml]: nit: terminal newline.

File: recipes/tempita/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  version: {{ version }}
+
+source:
+  fn: Tempita-{{ version }}dev.tar.gz
+  url: https://pypi.io/packages/source/T/Tempita/Tempita-{{ version }}.tar.gz
+  md5: 365c3b4f36435e2178902d5619301140
+
+build:
+  number: 0
+  script: python setup.py install --old-and-unmanageable
```
Comment (jakirkham) [inline recipes/tempita/meta.yaml]: Until we pick something different, could we please stick with the standard?

    python setup.py install --single-version-externally-managed --record=record.txt

**Comment by msarahan** [comment]
like cython, appveyor is all green, but github still says yellow.  Merge?

File: recipes/tempita/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/tempita/meta.yaml]: Done.

**Comment by jakirkham** [comment]
LGTM. Thanks @msarahan.

**Pull request merged**